runtimes/pythonrt/Makefile

```diff
@@ -15,7 +15,7 @@ test-py:
 
 # You can set the TESTOPTS to pass options to `go test`
 test-go:
-	golangci-lint run ./...
+	../../bin/tools/golangci-lint run ./...
```

Comment:
I prefer the `tools` directory to be in path and not use this.

Reply:
you want to set the path in this Makefile?

Reply:
Maybe? Not that important.